package.json

```diff
@@ -32,7 +32,7 @@
     "tslint-config-prettier": "^1.18.0",
     "tslint-plugin-prettier": "^2.3.0",
     "tslint": "^6.1.1",
-    "typescript": "^3.8.3"
+    "typescript": "^4.0.3"
```

Comment:
Does Stitches depend on Typescript v4+?

Asking for docs

Reply:
Yes we will depend on 4.1 now that it's a RC